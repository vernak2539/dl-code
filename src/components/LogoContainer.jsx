import React from "react";
import UserIcon from "./UserAccount";

export default () => (
  <div className="logo-container section">
    <UserIcon />
  </div>
);
