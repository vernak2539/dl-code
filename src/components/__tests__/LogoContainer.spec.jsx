import React from 'react';
import { shallow } from 'enzyme';
import LogoContainer from '../LogoContainer';

describe('Component > Logo Container', () => {
	it('should render correctly', () => {
		expect(shallow(<LogoContainer />)).toMatchSnapshot();
	});
});
