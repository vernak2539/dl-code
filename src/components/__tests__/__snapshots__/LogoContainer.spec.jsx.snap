// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component > Logo Container should render correctly 1`] = `
<div
  className="logo-container section"
>
  <Component />
</div>
`;
